docs/using-device-manager.md

```diff
@@ -0,0 +1,462 @@
+# Using DeviceManager
+
+Using DeviceManager is indeed simple: create a template with attributes and then create devices using that template. That's it.
+This page will show how to do that.
+
+All examples in this page consider that all dojot's components are up and running (check [the documentation](http://dojotdocs.readthedocs.io/) for how to do that). All request will include a ```${JWT}``` variable - this was retrieved from [auth](https://github.com/dojot/auth) component.
+
+## Creating templates and devices
+
+Right off the bat, let's retrieve a token from `auth`:
+
+```bash
+curl -X POST http://localhost:8000/auth \
+-H 'Content-Type:application/json' \
+-d '{"username": "admin", "passwd" : "admin"}'
+```
+
+```json
+{
+  "jwt": "eyJ0eXAiOiJKV1QiLCJhbGciOiJIU..."
+}
+```
+
+This token will be stored in ```bash ${JWT}``` bash variable, referenced in all requests.
+
+*IMPORTANT*: Every request made with this token will be valid only for the user associated with this token. For instance, listing created devices will return only those devices which were created by this user.
+
+-------------
+
+A template is, simply put, a model from which devices can be created. They can be merged to build a single device (or a set of devices). It is created by sending a HTTP request to DeviceManager:
+
+```bash
+curl -X POST http://localhost:8000/template \
+-H "Authorization: Bearer ${JWT}" \
+-H 'Content-Type:application/json' \
+-d ' {
+  "label": "SuperTemplate",
+  "attrs": [
+    {
+      "label": "temperature",
+      "type": "dynamic",
+      "value_type": "float"
+    },
+    {
+      "label": "pressure",
+      "type": "dynamic",
+      "value_type": "float"
+    },
+    {
+      "label": "model",
+      "type": "static",
+      "value_type" : "string",
+      "static_value" : "SuperTemplate Rev01"
+    }
+  ]
+}'
+```
+
+Supported value types are: "float", "string", "geo" (for georeference data)s, "integer". Supported value types are: "dynamic" and "static" (static attribute values defined here will be copied to all associated devices)
```

Comment:
perhaps?
```markdown
Supported `type` values are "dynamic", "static" and "meta". Supported `value_types` are "float", "geo", "string", "integer".
```

:thinking: both are actually strings, that would normally be interpreted by an iotagent, or another component (sometimes even an app).

Another important proposal was the usage of a "meta" type, so that system config is not mixed up with device data.